src/test/librbd/test_Utils.cc

```diff
@@ -0,0 +1,52 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "test/librbd/test_fixture.h"
+#include "test/librbd/test_support.h"
+#include "librbd/ImageCtx.h"
+#include "librbd/Utils.h"
+
+using librbd::util::data_object_name;
+
+void register_test_utils() {
+}
+
+struct TestUtils : public TestFixture {
+    librbd::ImageCtx *ictx;
+
+    void SetUp() override {
+      TestFixture::SetUp();
+      ASSERT_EQ(0, open_image(m_image_name, &ictx));
+    }
+
+    void TearDown() override {
+      close_image(ictx);
+      TestFixture::TearDown();
+    }
+
+    void expect_object_name(uint64_t object_no,
+                            const std::string& expected_object_name_postfix) {
+      EXPECT_STREQ(
+              (ictx->object_prefix + std::string(".") +
+               expected_object_name_postfix).c_str(),
+              data_object_name(ictx, object_no).c_str());
```

Comment:
No need to resort to C strings:
```suggestion
    EXPECT_EQ(data_object_name(m_ictx, object_no),                
              m_ictx->object_prefix + std::string(".") +                            
              expected_object_suffix);
```